weblate/urls.py

```diff
@@ -186,6 +186,12 @@
         weblate.lang.views.CreateLanguageView.as_view(),
         name="create-language",
     ),
+    path(
+        # Site-wide add-ons
+        "manage/addons/",
+        weblate.addons.views.AddonList.as_view(),
+        name="addons",
+    ),
```

Comment:
This is now duplicated with `name="manage-addons"` below. Are both really needed?